.classpath

```diff
@@ -5,25 +5,25 @@
 	<classpathentry excluding="BUILD.bazel" kind="src" path="gapic/res"/>
 	<classpathentry including="com/google/gapid/**" kind="src" path="bazel-bin/gapic/src/main"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
-	<classpathentry kind="lib" path="bazel-bin/core/data/pod/libpod_proto-speed.jar" sourcepath="bazel-genfiles/core/data/pod/pod_proto-speed-src.jar"/>
-	<classpathentry kind="lib" path="bazel-bin/core/image/libimage_proto-speed.jar" sourcepath="bazel-genfiles/core/image/image_proto-speed-src.jar"/>
-	<classpathentry kind="lib" path="bazel-bin/core/log/log_pb/liblog_pb_proto-speed.jar" sourcepath="bazel-genfiles/core/log/log_pb/log_pb_proto-speed-src.jar"/>
-	<classpathentry kind="lib" path="bazel-bin/core/os/device/libdevice_proto-speed.jar" sourcepath="bazel-genfiles/core/os/device/device_proto-speed-src.jar"/>
-	<classpathentry kind="lib" path="bazel-bin/core/stream/libstream_proto-speed.jar" sourcepath="bazel-genfiles/core/stream/stream_proto-speed-src.jar"/>
-	<classpathentry kind="lib" path="bazel-bin/external/com_google_protobuf/libprotobuf_java.jar"/>
+	<classpathentry kind="lib" path="bazel-bin/core/data/pod/libpod_proto-speed.jar" sourcepath="bazel-bin/core/data/pod/pod_proto-speed-src.jar"/>
+	<classpathentry kind="lib" path="bazel-bin/core/image/libimage_proto-speed.jar" sourcepath="bazel-bin/core/image/image_proto-speed-src.jar"/>
+	<classpathentry kind="lib" path="bazel-bin/core/log/log_pb/liblog_pb_proto-speed.jar" sourcepath="bazel-bin/core/log/log_pb/log_pb_proto-speed-src.jar"/>
+	<classpathentry kind="lib" path="bazel-bin/core/os/device/libdevice_proto-speed.jar" sourcepath="bazel-bin/core/os/device/device_proto-speed-src.jar"/>
+	<classpathentry kind="lib" path="bazel-bin/core/stream/libstream_proto-speed.jar" sourcepath="bazel-bin/core/stream/stream_proto-speed-src.jar"/>
+	<classpathentry kind="lib" path="bazel-bin/external/com_google_protobuf/libprotobuf_java.jar" sourcepath="bazel-external/com_google_protobuf/java"/>
```

Comment:
Should the sourcepath be a `.jar`, like all others around seem to be?

Reply:
No, in this case it's a source directory, since we build the protobuf runtime library from source.